daisy_workflows/image_test/metadata-script/metadata-script.subwf.json

```diff
@@ -0,0 +1,44 @@
+{
+  "Name": "metadata-script-test",
+  "Vars": {
+    "source_image": {"Required": true, "Description": "Image to be tested"},
+    "startup_hash": {"Required": true, "Description": "Hash of startup script"},
+    "shutdown_hash": {"Required": true, "Description": "Hash of shutdown script"},
+    "startup_msg": {"Required": true, "Description": "Startup script message to be verified when script exists"},
+    "shutdown_msg": {"Required": true, "Description": "Shutdown script message to be verified when script exists"},
+    "no_startup_msg": {"Required": true, "Description": "Startup script message to be verified when script is missing"},
+    "no_shutdown_msg": {"Required": true, "Description": "Shutdown script message to be verified when script is missing"}
```

Comment:
Are these variables going to be used in the future?

Reply:
what do you mean? This subworkflow has these variables set from `metadata-script-linux.wf.json` and `metadata-script-windows.wf.json`, which are the main workflows.
And they are all used below when it creates subworkflows.

Reply:
i don't know what I was looking at before, nvm